www/http/account.go

```diff
@@ -26,3 +29,36 @@ func (s *Server) GetAccountHandler(w http.ResponseWriter, r *http.Request) {
 
 	s.writeHTML(w, tm.html())
 }
+
+// GetAccountByNumberHandler returns a handler to get account by number.
+func (s *Server) GetAccountByNumberHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	num, err := strconv.ParseInt(vars["number"], 10, 32)
+	if err != nil {
+		s.writeError(w, err)
+		return
+	}
+
+	if num > math.MaxInt32 || num < math.MinInt32 {
```

Comment:
This check here is not necessary. HTTP is just a wrapper and should have not logic.